suite/tests/api/ibl-stress.c

```diff
@@ -37,10 +37,19 @@
 #include "thread.h"
 #include "condvar.h"
 
-#ifdef LARGER_TEST
+#if defined(LARGER_TEST)
 /* 20K sequences gives us ~150K bbs. */
 #    define NUM_SEQUENCES 20000
 #    define NUM_THREADS 16
+#elif defined(TEST_FAR_LINK_AARCH64)
+/* These values trigger far fragment linking path on AArch64. With this option,
+ * the test is too big to enable in CI. The following changes are required to
+ * the regular ctest options:
+ * - set timeout to 15 mins, instead of the default 90 seconds
```

Comment:
You can have these changes automatically made, right?  No reason to require manual changes.

Reply:
The current support for marking tests as short/long seems to only allow applying different DR options, and it's tricky to skip long tests in the regular suite. https://github.com/DynamoRIO/dynamorio/blob/eb57ba9555cfd19fa6889a2ba764aba9a524d39d/suite/tests/CMakeLists.txt#L99.

So I created a new instance of `ibl-stress` that is compiled with `-DTEST_FAR_LINK_AARCH64`. I marked it with `_LONG` suffix and added logic to skip such marked long tests in `is_test_enabled_by_name`, based on the `TEST_LONG` flag.

Reply:
Yes I think there was no prior concept of labeling an individual test.  Looks like a good solution.